broker/src/main/java/org/apache/rocketmq/broker/processor/AbstractSendMessageProcessor.java

```diff
@@ -173,13 +173,10 @@ protected RemotingCommand msgCheck(final ChannelHandlerContext ctx,
             return response;
         }
 
-        if (TopicValidator.RMQ_SYS_SCHEDULE_TOPIC.equals(requestHeader.getTopic())) {
-            response.setCode(ResponseCode.NO_PERMISSION);
-            response.setRemark("sending message to topic[" + requestHeader.getTopic() + "] is forbidden");
+        if (!TopicValidator.validateTopic(requestHeader.getTopic(), response)) {
             return response;
         }
-
-        if (!TopicValidator.validateTopic(requestHeader.getTopic(), response)) {
+        if (!TopicValidator.validateBlacklistTopic(requestHeader.getTopic(), response)) {
```

Comment:
what is a blacklist? It's just not allowed to send a message to this topic. It would be better to name it notAllowedSending.

Reply:
ok，I'll change it